### PR TITLE
add bot 7000104432  onregularinvesting.calculator

### PR DESCRIPTION
**Pull request by imlidian**
Simulate the benefits of historical fixed investment and reunderstand past investment decisions.
To improve current investment decisions by revising their past erroneous concepts.

### CODE REVIEW

File: src/static/bots/7000104432/index.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    "client_id": "67bae580-e0d7-4286-a06e-9f970698bad2",
+    "url": "https://calc.imlidian.com/",
+    "name": {
+      "en": "onregularinvesting.calculator",
+      "zh": "定投计算器"
+    },
+    "text": {
+      "en": ["onregularinvesting.calculator", "simulate the benefits of historical fixed investment and reunderstand past investment decisions", 
+      "To improve current investment decisions by revising their past erroneous concepts."],
+      "zn": ["定投计算器", "模拟历史定投收益,重新理解过往投资决策", "改善当下决策质量"]
```
Comment (lyricat) [inline src/static/bots/7000104432/index.json:12]: should be "zh", not "zn"

File: src/static/bots/7000104432/index.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    },
+    "category": "Utilities",
```
Comment (lyricat) [inline src/static/bots/7000104432/index.json:14]: should be "util", FYI: https://github.com/fox-one/mixin.fan/blob/master/src/bots.ts#L66

File: src/static/bots/7000104432/index.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    "tags": {
+      "en": ["calculator", "invest","onregularinverting"],
+      "zh": ["定投", "计算器","定投计算器"]
+    },
+    "language": "zh",
+    "languages": ["en","zh"],
+    "developer": {
+      "name": "lidian",
+      "url": "https://www.imlidian.com"
+      "support_url": "https://github.com/imlidian"
```
Comment (lyricat) [inline src/static/bots/7000104432/index.json:24]: support_url should not under the "developer". Please refer to other bot's config file https://github.com/fox-one/mixin.fan/blob/master/src/static/bots/7000000014/index.json

**Review by lyricat — CHANGES_REQUESTED** [review]

File: src/static/bots/7000104432/index.json (hunk 1 above)
Comment (imlidian) [reply src/static/bots/7000104432/index.json]: done

File: src/static/bots/7000104432/index.json (hunk 2 above)
Comment (imlidian) [reply src/static/bots/7000104432/index.json]: done

File: src/static/bots/7000104432/index.json (hunk 3 above)
Comment (imlidian) [reply src/static/bots/7000104432/index.json]: done thx a lot

**Pull request merged**